### ISSUE TITLE
Jetpack: Different Jetpack connection experience bewteen notification and stats

### ISSUE THREAD

**Issue by aerych** [wordpress-mobile/WordPress-Android#3275]
There are differences between stats and notifications in the experience connecting to Jetpack for a logged out user. Stats takes the user directly to a sign in screen. I'm not sure if its assuming the user has a wpcom account, or if knows there is a wpcom account.  I thought it would be handy to show an option to create an account regardless.  The experience from Notifications has the option to create an account after tapping "sign in".

![screenshot_2015-09-28-19-16-07](https://cloud.githubusercontent.com/assets/1435271/10152264/00bdb18c-6616-11e5-8404-7dffdc2863e7.png)

![screenshot_2015-09-28-19-16-21](https://cloud.githubusercontent.com/assets/1435271/10152263/00bced60-6616-11e5-8758-42679bccfeaf.png)

**Comment by maxme** [comment]
> I'm not sure if its assuming the user has a wpcom account, or if knows there is a wpcom account

I think this only happens when the self hosted blog has the jetpack plugin installed (but that doesn't guarantee the user accessing that screen has JP credentials).

**Comment by daniloercoli** [comment]
That's right. This screen is only shown in Stats when the self-hosted blog has the Jetpack plugin already installed, but the user is not logged to wpcom, or the main wpcom account isn't linked with the blog.

**Comment by daniloercoli** [comment]
Should we keep this ticket open?

**Comment by aerych** [comment]
Up to you guys. My 2 cents is the experience should be the same but :shrug:

**Comment by daniloercoli** [comment]
I think Notifications uses that ("placeholder") version of the login screen since it is shown as a Tab in the main tabbed interface. Tapping "sign in" should open a modal window. cc @roundhill

**Comment by maxme** [comment]
We should have a coherent approach everywhere in the app:
* if the site has jetpack installed, and rest API enabled: ask the user to sign in (hide the "create account button")
* if the site has jetpack installed, rest API disabled (or anything else that prevent jetpack use): tell the user something is wrong and they should fix it on their site by doing this or that.
* if the site has jetpack installed but the user is logged in another wpcom account: tell the user we only support one account in the app and they can't use jetpack features on that site.
* if the site doesn't have jetpack installed: tell the user they have to install jetpack to use this feature, and eventually open a logged in Webview so they can install Jetpack easily.

**Comment by hoverduck** [comment]
Related: https://github.com/wordpress-mobile/WordPress-Android/issues/2646
